src/Integration.UnitTests/Binding/BindingWorkflowTests.cs

```diff
@@ -301,98 +300,6 @@ public void BindingWorkflow_InstallPackages_Cancellation()
         }
 
         [TestMethod]
```

Comment:
just add a link to this commit on this Jira ticket to add the support of AdditionalFiles back, it will be straightforward to revert